gems/smithy-client/lib/smithy-client/shapes.rb

```diff
@@ -68,12 +68,12 @@ def initialize(options = {})
           @members = {}
         end
 
-        # @return [Hash<String, MemberShape>]
+        # @return [Hash<Symbol, MemberShape>]
         attr_accessor :members
 
         # @return [MemberShape]
         def add_member(name, shape, traits)
-          @members[name] = MemberShape.new(name, shape, traits)
+          @members[name] = MemberShape.new(name.to_s, shape, traits)
```

Comment:
You should not need to convert symbol to string for this to work.

Reply:
Per our discussion - previously, i was thinking that we need a way to reference the `name` back but thinking a bit further, this is not necessary. Might be useful if its a unique member shape id or something but that's not quite it so moved to remove these. (always could go back to add them IF we need them)